src/java.base/share/classes/javax/crypto/Cipher.java

```diff
@@ -181,22 +181,22 @@ public class Cipher {
         Debug.isOn("engine=") && !Debug.isOn("cipher");
 
     /**
-     * Constant used to initialize cipher to encryption mode.
+     * Constant used to initialize {@code Cipher} to encryption mode.
```

Comment:
If it's lower case, we normally just leave it? Not sure what is the criteria for changing this.

Reply:
Personally, I'd against making this change, i.e. changing all "cipher" occurrences to "{@code Cipher}". Can we just stick to fixing what's broken?

Reply:
In this case the sentence was about initializing a cipher object, so I changed it to `{@code Cipher}`. If a sentence says "cipher block size" then I tried to leave it alone because it's referring to an algorithm rather than a class or object. I could add "object" after `{@code Cipher} `if it's not present. That would make it more consistent.

Reply:
Ok, add object afterwards would make it more consistent.

Reply:
fixed: `{@code Cipher} `object